Automattic/vip-go-mu-plugins
# Add es-wp-query subtree for Search

## Description

Adds automattic/es-wp-query as a subtree under `search` so we can use it for query offloading if it hasn't been included already.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant unit tests (if applicable).
- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

Nothing to test yet as this code isn't hooked in anywhere, but worth doing a checkout to ensure the subtree is correctly populated.
